http-filter-example/http_filter.h

```diff
@@ -0,0 +1,30 @@
+#include <string>
+
+#include "server/config/network/http_connection_manager.h"
+
+namespace Envoy {
+namespace Http {
+
+const LowerCaseString headerKey = LowerCaseString("via");
```

Comment:
This should probably be a class constant, otherwise if this header is imported into distinct files, you'll end up defining the same symbol twice and end up with linker errors.